src/tribol/geom/ContactPlane.cpp

```diff
@@ -2275,7 +2287,9 @@ bool ContactPlane2D::computeLocalInterpenOverlap()
    // Debug check the number of interpenetrating vertices
    if (k > 2)
    {
-      SLIC_ERROR("ContactPlane2D::computeLocalInterpenOverlap more than 2 interpenetrating vertices detected.");
+      SLIC_INFO("ContactPlane2D::computeLocalInterpenOverlap() more than 2 interpenetrating vertices detected.");
```

Comment:
Same question here -- should this be a `SLIC_INFO` and, if so, should it have more context about the problem?

Reply:
Fixed.